tests/core/featurizers/test_tracker_featurizer.py

```diff
@@ -1,17 +1,33 @@
-from typing import Text
+from typing import Text, Dict, List, Optional
```

Comment:
Before anyone reviews the changes to this file. We know that there is some amount of redundancy in the tests added here and hence there is already a [follow up issue](https://github.com/RasaHQ/rasa/issues/8798) to refactor them. It wasn't considered release blocking and hence will be done afterwards by the feature's team.